history-of-the-quakers-v4/original/22-book-7-appendix-2.adoc

```diff
@@ -501,12 +521,20 @@ and obstruct the business of the ensuing treaty at Easton,
 where the complaints made by them will be fully and impartially heard,
 and I hope finally accommodated.
 
-William Denny
+[.signed-section-signature]
+William Denny.
 
+[.signed-section-context-close]
 To Mr. Israel Pemberton, and the other Gentlemen.
```

Comment:
So this just repeats what the [.letter-heading] says, but I didn't know if I should have taken it out or not.